pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = -p nengo.tests.options --simulator nengo_loihi.Simulator --neurons nengo.LIF,nengo.SpikingRectifiedLinear
+addopts = -p nengo.tests.options --simulator nengo_loihi.Simulator --ref-simulator nengo_loihi.Simulator --neurons nengo.LIF,nengo.SpikingRectifiedLinear
```

Comment:
Do we expect to pass all the RefSimulator tests? Originally, I think this was designed for backends that use the full Nengo builder (like nengo_dl and nengo_ocl). But maybe we're able to pass all these tests...

Reply:
I did run the test suite to verify that everything passed.